packages/svelte-vscode/README.md

```diff
@@ -74,6 +74,40 @@ It's also necessary to add a `type="text/language-name"` or `lang="language-name
 </style>
 ```
 
+### Troubleshooting
+
+#### I get weird type errors on my html tags
+
```

Comment:
Might be worthwhile adding an example to enhance understanding. I think the screenshot in #228 would suffice